performance_manager/lib/postgres_schema.py

```diff
@@ -11,7 +11,7 @@ class VehiclePositionEvents(SqlBase):  # pylint: disable=too-few-public-methods
 
     __tablename__ = "eventsVehiclePositions"
 
-    pk_id = sa.Column(sa.Integer, primary_key=True)
+    pk_id = sa.Column(sa.Integer, primary_key=True, index=True)
```

Comment:
what does `index=True` do here?

Reply:
It tells the database to create an index on the Column, I kind of assumed that the `primary_key` flag would force an index creation, but I didn't see an index for these columns in pgadmin when I was looking at the tables.